### DISCUSSION TITLE
Zmiana skryptów, aby dla nieprzetłumaczonych fragmentów przekierowywać (linkować) do wersji angielskiej.

### DISCUSSION THREAD

**Discussion by speplinski** [irdpl/wcag#9] [Organizacja pracy]
Myślałem o tej zmianie o której wczoraj rozmawialiśmy @zwiastunsw i po analizie widzę sporo problemów z tym podejściem.

Gdyby rzeczywiście oprzeć wszystko o GitHub (jako fork z w3c/wcag), to nie trzeba by zmieniać linków do nieprzetłumaczonych fragmentów, tylko synchronizować repozytorium. Dzięki temu byłaby również możliwość na pracy nad innymi sekcjami (jak np. understanding) fragmentarycznie.

**Comment by zwiastunsw** [comment]
Polegam na Twojej znajomości rzeczy. Im większa elastyczność i możliwości synchronizacji (oraz obserwowania/raportowania zmian dokonanych w oryginale), aby można je dyskontować w rzeczach przetłumaczonych, tym lepiej.  
Potrzebujemy, tak myślę, platformy, która posłuży nam do spolonizowania WCAG w takim zakresie, na jaki tylko znajdziemy siły. Samo i szybko się nie zrobi, a powoli i systematycznie jest szansa że za rok, dwa lub pięć będziemy mieć kompletne WCAG (nie tylko dokument podstawowy)
